app/Http/Controllers/VideoSessionFilesController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\VideoSessionFileCreateRequest;
+use App\Http\Resources\VideoSessionFileResource;
+use App\Utils\UploadImage;
+use App\Models\File;
+use App\Models\VideoSessionFile;
+use App\Models\ProductDetailVideo;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Storage;
+
+class VideoSessionFilesController extends Controller
+{
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \App\Http\Requests\VideoSessionFileCreateRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(VideoSessionFileCreateRequest $request)
```

Comment:
Insert into `ProductFIle` and check if product relation exist unless raise an error